templates/environment/partials/environment-manager-role.yml

```diff
@@ -74,6 +74,7 @@ EnvironmentManagerRole:
             "cloudformation:Describe*",
             "cloudformation:DetectStackDrift",
             "cloudformation:DetectStackResourceDrift",
+            "cloudformation:ListStacks",
```

Comment:
Are we using this action? I thought we only use `DescribeStacks`.

Reply:
same